app/services/solr_document_metadata.rb

```diff
@@ -35,6 +35,10 @@ def solr_lookup(key)
     document[key]
   end
 
+  def container(_key = nil)
```

Comment:
Cool, I'm so pleased the way I reworked this class was immediately useful for another issue!!